index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="css/my_portfolio.css">
+</head>
+<body>
+    <header id="header" class="header"> 
+        <div class="logo">John Doe</div>
+        <div class="burger"><img src="assets/images/burger.png" alt="" class="d-none-lg"></div>
+        <nav id="navbar" class="d-none d-block-lg">
+            <ul class="nav-list d-flex-lg list-style-none">
+                <li class="nav-item">
+                  <a href="#" class="nav-link">Portfolio</a>
+                </li>
+                <li class="nav-item">
+                  <a href=".about-us" class="nav-link">About</a>
+                </li>
+                <li class="nav-item">
+                  <a href="#" class="nav-link">Contact</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-link"><img src="assets/images/message-box.png" alt=""></a>
+                </li>
+              </ul>
+        </nav>
+    </header>
+
+    <section class="my-self">
```

Comment:
- [ ] Hi, this `section` should be laid out using `flexbox`. the purpose of this is to get you comfortable with laying out designs using flexbox. Thanks. please make this change.